internal/impl/slack/input.go

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2025 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+package slack
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Jeffail/shutdown"
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/socketmode"
+)
+
+func init() {
+	err := service.RegisterInput("slack", inputSpec(), newInput)
+	if err != nil {
+		panic(err)
+	}
+}
+
+const (
+	iFieldAppToken = "app_token"
+	iFieldBotToken = "bot_token"
+)
+
+func inputSpec() *service.ConfigSpec {
+	return service.NewConfigSpec().
```

Comment:
Missing docs about metadata

Reply:
It's in there

Reply:
Oops... My bad. I was looking for the typical pattern. Never mind 🙈

Reply:
yeah was certainly lazy and am relying on Angela to make it right